starter-code/src/functions-and-arrays.js

```diff
@@ -11,30 +27,89 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
```

Comment:
Strictly speaking you could , instead of declaring the variable lengthOfWord use array[i].length in the conditional statement. This will make your code slightly shorter and more readable. Nevertheless, good work!